assets/scss/app.scss

```diff
@@ -1,13 +1,12 @@
 @import "hof/frontend/themes/gov-uk/styles/govuk";
-@import "govuk-frontend/govuk/components/tabs/tabs";
+@import "node_modules/govuk-frontend/govuk/all";
```

Comment:
it may be you can remove this package altogether as you are using the gov-uk style from the hof frontend theme. Not sure why you need to be using both of these. Generally the hof framework serves up the frontend for us so it seems like we are adding the gov-uk frontend npm mpdule unnecessarily

Reply:
I think it needs a referencing, if I remove it then this happens
![Screen Shot 2022-06-30 at 12 20 06](https://user-images.githubusercontent.com/12494656/176665227-311050e7-3361-4870-9d86-b7ef52fca698.png)

Then I tried pulling it in via node_modules from hof i.e. like this @import `"node_modules/govuk-frontend/govuk/all"; `
but it can't find anything in node_modules
![Screen Shot 2022-06-30 at 12 25 06](https://user-images.githubusercontent.com/12494656/176667303-f815697d-49a0-4b31-b8b0-1d77fe1f6ea2.png)

Happy to try something else

Reply:
cool, how would we reference it in the hof framework? We have the line `hof/frontend/themes/govuk-styles/govuk` which is pulled in is because it's copied from an old govuk package straight into hof I believe.  That line is every hof package it seems

Reply:
You're advice worked a treat, it's now in hof and no longer in rotm.  Thanks man 👍 😄